app/services/processors/blog_posts_updater.rb

```diff
@@ -24,9 +23,12 @@ def categorizer
       @categorizer ||= BlogPostCategorizer.new
     end
 
-    def technologies_for(blog_post_id)
-      blog_post_payload = wordpress_service.blog_post(blog_post_id)
+    def technologies_for(blog_post)
+      blog_post_payload = wordpress_service.blog_post(blog_post.blog_id)
       categorizer.technologies_for(blog_post_payload)
+    rescue Faraday::Error => exception
+      ExceptionHunter.track(exception)
+      blog_post.technologies
```

Comment:
I'm not sure what this line is for.

Reply:
It's because we care about the return value of this method. It should return the technologies that match a blog post, so that we can assign them to it (check [here](https://github.com/rootstrap/rs-code-review-metrics/pull/291/files#diff-a9bfba1ad10a239e09f1b608d16cd788R10))
If we couldn't get the information from the API to know which ones we should assign, at least we will be assigning the ones the blog post already had, allowing them to remain untouched.
So if the blog post had already technologies associated, they'll remain the same.
And if the blog post is new, we'll leave it without any technology (we could assign the `other` technology in this case, though).

Reply:
Makes total sense 👍 my bad